integration-libs/s4-service/checkout/components/checkout-service-details/checkout-service-details.component.ts

```diff
@@ -59,13 +59,9 @@ export class CheckoutServiceDetailsComponent implements OnInit, OnDestroy {
     this.subscription.add(
       this.selectedServiceDetails$.subscribe((selectedServiceDetails) => {
         if (selectedServiceDetails && selectedServiceDetails !== '') {
-          const scheduledAt =
-            this.checkoutServiceSchedulePickerService.convertDateTimeToReadableString(
```

Comment:
@anjana-bl Please check this is in checkoutorder

Reply:
@vigneshkrish8691 I had checked this along with @anjana-bl . 

`this.checkoutServiceSchedulePickerService.convertDateTimeToReadableString(selectedServiceDetails)` was converting date differently on mac and on windows, so removed the statement.

The next statement `this.checkoutServiceSchedulePickerService.getServiceDetailsFromDateTime(selectedServiceDetails)` correctly interprets the date and time required for the form.